e2e/condition_catalog_service_test.go

```diff
@@ -529,6 +529,20 @@ task {
 	assert.Equal(t, 2, initCount)
 	initResources := filepath.Join(tempDir, initTask, resourcesDir)
 	validateServices(t, true, []string{sharedService}, initResources)
+
+	// Register a new service that will trigger the created task
+	service = testutil.TestService{ID: "web-test-new", Name: "web-test-new", Tags: []string{"tag_b"}}
+	testutils.RegisterConsulService(t, srv, service, defaultWaitForRegistration)
+
+	// Check that created task does trigger
+	api.WaitForEvent(t, cts, taskName, registrationTime, defaultWaitForEvent)
+	count = eventCount(t, taskName, cts.Port())
+	expectedCount++
+	assert.Equal(t, expectedCount, count)
+	resourcesPath := filepath.Join(tempDir, taskName, resourcesDir)
+	validateServices(t, true, []string{"services_" + sharedService}, resourcesPath)
```

Comment:
I noticed there's a `"services_"` here unlike previous `validateServices()`. I wasn't able to figure out the context for it. Would you mind staring?

Reply:
That's due to the module itself adds this in front of the file name for the service! I don't think we check this very often so I can leave a comment here and for the tags file!

Reply:
Ohhh gotcha. Thank you for explaining that to me. Makes sense.